### ISSUE TITLE
Low font quality when resizing the window size

### ISSUE THREAD

**Issue by Kasterisk** [Revolutionary-Games/Thrive#1852]
In general the font quality get messed up if you resize the window to smaller/(bigger?) size in windowed mode, for instance some of them would get blurry with filter enabled, but disabling it could also make the texts have this weirdness:
![2020-11-19_12 46 43 3914](https://user-images.githubusercontent.com/54026083/99626723-65a04d80-2a65-11eb-94b3-e2439a6285cd.png)
Maybe the UI scaling doesn't work well with fonts? either way, would be great to fix this.

**Comment by hhyyrylainen** [comment]
For me the fonts resize just fine without blurriness even when going to a small window size and back, and also weird aspect ratios. Might be a Windows only issue.

**Comment by RacerBG** [comment]
I experience the same things as @Kasterisk without Vsync and Anti-aliasing but even turning them on doesn't really help.

![Untitled](https://user-images.githubusercontent.com/68707872/99724110-cfe3dd00-2abb-11eb-9765-a62ddc818d0a.png)

**Comment by RacerBG** [comment]
This looks better now (on master and in 0.5.8.1), at least for me. @Kasterisk ?

**Comment by Kasterisk** [comment]
Yes, font filtering has been enabled for quite a long time now which removes the jaggedness. Still, it's only halfway there and is not quite the crispy clear font quality that we want. Basically has to wait Godot 4 for this.
